modules/cluster/irsa.tf

```diff
@@ -421,7 +421,7 @@ module "iam_assumable_role_secrets-secrets-manager" {
 // ----------------------------------------------------------------------------
 // External Secrets - Parameter Store
 // ----------------------------------------------------------------------------
-data "aws_iam_policy_document" "parameter-store-policy" {
+data "aws_iam_policy_document" "system-manager-policy" {
```

Comment:
Why this change? since the other one is called secrets manager, and outside system manager do we have to make this change?

Reply:
so this change is purely to be uniform with the rest of external secrets. external-secrets calls it "systemManager", so just so it's the same everywhere, I want to change it here too